quattor/types/aquilon/system.pan

```diff
@@ -112,7 +114,7 @@ type structure_personality = {
     "description"   ? string
     "class"         ? string with match(SELF, '(INFRASTRUCTURE|APPLICATION)')
     "users"         ? string[]
-    "systemgrn"     : string[]
+    "systemgrn"     ? string[]
```

Comment:
Recent versions of the broker no longer generate this but I guess you need it for people on old broker versions?

Reply:
No, I think we should require the last version of the broker for people wanting to use this version of the template library. I think we are only 3 sites running a broker and two of us probably up-to-date! Fixed.